src/libs/actions/Report.ts

```diff
@@ -625,13 +625,6 @@ function openReport(
         parameters.shouldRetry = false;
     }
 
-    const report = ReportUtils.getReport(reportID);
-    // If we open an exist report, but it is not present in Onyx yet, we should change the method to set for this report
-    // and we need data to be available when we navigate to the chat page
-    if (isEmptyObject(report)) {
-        optimisticData[0].onyxMethod = Onyx.METHOD.SET;
-    }
-
```

Comment:
This is to fix all the found issues from our PR 
https://github.com/Expensify/App/issues/37247, https://github.com/Expensify/App/issues/37234, https://github.com/Expensify/App/issues/37300, and https://github.com/Expensify/App/issues/37251.

The root cause is explained [here](https://github.com/Expensify/App/issues/37247#issuecomment-1965912623)